mailbox/lucene/src/main/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndex.java

```diff
@@ -1068,63 +1068,53 @@ private SortField createSortField(SearchQuery.Sort s, boolean reverse) {
             case Arrival:
                 if (reverse) {
                     return ARRIVAL_MAILBOX_SORT_REVERSE;
-                } else {
-                    return ARRIVAL_MAILBOX_SORT;
                 }
+                return ARRIVAL_MAILBOX_SORT;

```

Comment:
Seriously there are place where ternary expression are good and there are this place. But we are not going to discuss that again we already this. But I still not agree with you one that.